File: Scala/4. Median of Two Sorted Arrays.scala
object Solution {
  def findMedianSortedArrays(nums1: Array[Int], nums2: Array[Int]): Double = {
    val com_Array = (nums1 ++ nums2).sorted
    if(com_Array.size % 2 != 0){
      return com_Array((com_Array.size / 2))
    }
    else{
      val pos1 = com_Array.size / 2
      return ((com_Array(pos1) + com_Array(pos1 - 1).toDouble) / 2)
    }
  }
}

File: Python/4. Median of Two Sorted Arrays.py
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        com_List = sorted(nums1 + nums2)
        if len(com_List) % 2 != 0:
            pos = int(len(com_List) / 2)
            return float(com_List[pos])
        else:
            pos1 = int(len(com_List) / 2)
            pos2 = pos1 - 1
            return float(com_List[pos1] + com_List[pos2]) / 2
